addon/webextension/background/main.js

```diff
@@ -97,6 +97,15 @@ window.main = (function () {
         title: browser.i18n.getMessage("notificationLinkCopiedTitle"),
         message: browser.i18n.getMessage("notificationLinkCopiedDetails", pasteSymbol)
       });
+    } else {
+      catcher.watchPromise(browser.tabs.query({active: true, currentWindow: true})
+      .then((tabs) => {
+        for (let tab of tabs) {
+          if(tab.url.match(/\/creating/i)) {
+            catcher.watchPromise(browser.tabs.remove(tab.id));
```

Comment:
Thanks for the review @dannycoates !
I tried adding a second function parameter to `then`, but wasn't very sure about how to break out of the Promise chain because the promise was still getting resolved (possibly because the `tabs.remove` returns a promise that is resolved if the tab is closed). It was still showing a **copied to clipboard** notification. I added a `catch` to the chain, and it seems to solve the problem. Let me know what you think!